spec/factories.rb

```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
+  factory :ingredient_mention do
+    
```

Comment:
Alphebatize this alongside the other factories, after sequences. Please also create the factory